docs/release-notes/2.1.0.md

```diff
@@ -0,0 +1,24 @@
+# Release Notes - 2.1.0
+## Features
+
+1.  Ability to specify an existing Azure virtual network prior to deployment [(#133)](https://github.com/microsoft/CromwellOnAzure/pull/133)
+2.  Documentation updates [(#143)](https://github.com/microsoft/CromwellOnAzure/pull/133), [(#144)](https://github.com/microsoft/CromwellOnAzure/pull/133)
+
+## Bug fixes and enhancements
+
+1.  Occassionally Azure Batch nodes are unusable, which can result in running nodes that are not terminated [(#142)](https://github.com/microsoft/CromwellOnAzure/pull/142)
+2.  Occassionally Cromwell cannot start, and the error in the logs is: "Could not acquire change log lock" [(#129)](https://github.com/microsoft/CromwellOnAzure/pull/129)
+3.  Occassionally workflows can fail due to transient Azure issues; retry policies have been added [(#140)](https://github.com/microsoft/CromwellOnAzure/issues/140)
+4.  When the Cromwell API is unavailable, the Trigger Service sets the workflow to failed, instead of only when Cromwell reports it failed or not found [(#147)](https://github.com/microsoft/CromwellOnAzure/pull/147)
+5.  Additional logging added to `mount_containers.sh` [(#131)](https://github.com/microsoft/CromwellOnAzure/pull/131)
+6.  Azure CosmosDB is set to autoscale on deployment.  A breaking change in the Azure CosmosDB API resulted in a regression that did not enable autoscale. [(#137)](https://github.com/microsoft/CromwellOnAzure/pull/137)
```

Comment:
Simplify to  just second sentence? e.g. A breaking change in the Azure CosmosDB API resulted in a regression that did not enable autoscale by default